conf/application.conf

```diff
@@ -79,7 +74,7 @@ metrics {
 # Microservice specific config
 
 auditing {
-  enabled = false
+  enabled = true
```

Comment:
leave this disabled locally - it should be disabled in our other services as well locally but don't worry about that we can address in other PRs

Reply:
Done.